### ISSUE TITLE
Need a way to create a Layout from the port of a node

### ISSUE THREAD

**Issue by harshaktg** [antvis/layout#134]
It is clear that the model object can specify nodes and edges, and when the layout is created from it, it works perfectly.
But is there a way to specify ports within the nodes - and then also specify edges that connects separate ports.

Can this be solved?

**Comment by zzzhanga** [comment]
Yes, I also think there is a need for a function to recalculate the connection method of the port. The current layout only changes the position of the nodes
